pkg/lib/userexport/service.go

```diff
@@ -22,6 +29,60 @@ func NewLogger(lf *log.Factory) Logger {
 }
 
 func (s *UserExportService) ExportRecords(ctx context.Context, request *Request) *Response {
-	// TODO: Add query logic
-	return nil
+	var total_user uint64 = 0
```

Comment:
No need to count the number of users. Remove this.

Reply:
removed the total count